src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * The Past Recent View.
+ *
+ * @package Tribe\Events\Views\V2\Views
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+use Tribe__Events__Main as TEC;
+use Tribe__Events__Rewrite as TEC_Rewrite;
+use Tribe__Utils__Array as Arr;
+use Tribe\Events\Views\V2\Utils;
+
+class Recent_Past_View extends View {
+	use List_Behavior;
```

Comment:
:no_entry_sign: Missing doc comment for class Recent_Past_View

<sub>hash: bf49437c46abd745ddaaee83a0856361</sub>